src/wallet/rpc/spend.cpp

```diff
@@ -1048,6 +1056,15 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
             coin_control.m_signal_bip125_rbf = options["replaceable"].get_bool();
         }
         SetFeeEstimateMode(*pwallet, coin_control, conf_target, options["estimate_mode"], options["fee_rate"], /*override_min_fee=*/false);
+
+        // Prepare new outputs by creating a temporary tx and calling AddOutputs().
+        if (!options["outputs"].isNull()) {
+            CMutableTransaction tempTx;
+            AddOutputs(tempTx, options["outputs"]);
+            for (const auto& output : tempTx.vout) {
+                outputs.push_back(output);
+            }
```

Comment:
In 41464504b74690dac173f0615f8fb2b444d8f0e9 "wallet: add `outputs` arguments to `bumpfee` and `psbtbumpfee`"

nit: Could be simplified to copying `tempTx.vout`

```suggestion
            outputs = tempTx.vout
```